Tests/StreamChatUITests/SnapshotTests/ChatThread/ChatThreadVC_Tests.swift

```diff
@@ -99,4 +99,77 @@ final class ChatThreadVC_Tests: XCTestCase {
         XCTAssertTrue(vc.messageListVC.listView is TestMessageListView)
         XCTAssertTrue(vc.messageComposerVC.composerView is TestComposerView)
     }
+
+    // MARK: - chatMessageListVC(_:footerViewForMessage:at)
```

Comment:
The snapshots showing "No replies" but actually having replies are a bit misleading. Shouldn't we actually have tests for whenever the thread is empty, where the footer should not be shown?

Reply:
The footer (at least on the React Native app) is visible even when there are no replies. I'll also discuss it with the designer team. 

We have tests that show the empty state `test_emptyAppearance`. 

I'll go ahead and update the tests that mistakenly show `NO REPLIES` when there are some

Reply:
Hm, tests like https://github.com/GetStream/stream-chat-swift/pull/2509/files#diff-7e8c52136701589324f567b355d9e0ebd7e7f073cf0b058813fa794e68dd2165 are a bit misleading, the "No replies" shouldn't be displayed there.